tutorials/rendering/multiple_resolutions.rst

```diff
@@ -328,6 +328,22 @@ that have a pixel art UI, so that the visible area in the 3D viewport doesn't
 reduce in size (which occurs when using ``canvas_items`` or ``viewport`` stretch
 mode with the ``integer`` scale mode).
 
+.. tip::
```

Comment:
The information here is good, but it almost feels like it should be it's own section on this page, and have the bit at the end about integer scaling remain a tip in this section. I get that fullscreen vs exclusive fullscreen isn't fully about multiple resolutions but there's no other place to put it really belongs in the docks.

Regardless I'd be ok with merging as is.